src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -1413,12 +1412,19 @@ public static void fullyRemoveCell(Design design, Cell cell, Map<Net, Set<SitePi
         cell.getParentCell().removeCellInst(cell.getEDIFCellInst());
     }
 
-    private static void handlePinRemovals(SitePinInst spi, Map<Net,Set<SitePinInst>> deferRemovals) {
-        boolean preserveOtherRoutes = true;
+    /**
+     * Helper method for either removing (and unrouting) a SitePinInst immediately (when deferRemovals is null)
+     * or deferring its removal by putting it into the deferRemovals map.
+     * @param spi SitePinInst object to be removed/unrouted.
+     * @param deferRemovals Optional map for deferring the removal of SitePinInst objects, grouped by their
+     *                      associated Net object.
+     */
+    public static void handlePinRemovals(SitePinInst spi, Map<Net,Set<SitePinInst>> deferRemovals) {
```

Comment:
```suggestion
    public static void handlePinRemovals(SitePinInst spi, Map<Net, Set<SitePinInst>> deferRemovals) {
```

Reply:
Oops sorry, missed this before pulling the trigger.